content/francineblanc/how-to-use-git-for-beginners.md

```diff
@@ -0,0 +1,280 @@
+
+![git-init](git-init.png)
+
+When `git init` is run inside the **Git Tutorial** folder, an empty Git repo is created inside of that directory. The repo is currently empty because Git doesn't know at this point which files it should track. Any untracked files can be viewed using the command:
+
+```bash
+git status
+```
+
+This will display untracked files in <span style="color:red">red</span>, and will show some useful output confirming that nothing has been added. For example, if `git status` is run in the **Git Tutorial** folder, which now contains a file called **learning-git.txt** the output looks like this:
+
+![git-status-untracked](git-status-untracked-files.png)
+
+The output will begin with "On branch main" which can be ignored for now. One of the files listed is one called **.DS_Store** that is automatically created by Mac OS X. This file contains information about system configurations, so should not be committed as part of the Git workflow. To make sure of this, a file called **.gitignore** should be made in the working directory.
+
+### .gitignore
+
+The **.gitignore** file is a special file which generally lives in the root of the working directory. It contains a list of files and/or directories which a developer has added that Git should not track, and so will not be committed. These files may be log files, or particular modules in a project that shouldn't be tracked, or (for Mac users) files like `.DS_Store`. Make sure to try and remember to add files which should not be tracked ***before*** they are committed.
+
+Now, with the **.gitignore file** made and the **.DS_Store** file added, running `git status` in the terminal shows the following output:
+
+![git-status-untracked-with-gitignore](git-status-untracked-with-ignore.png)
+
+## Updating the staging area
+
+In order for Git to start tracking changes, files need to be added to the staging area. This can be done with the command:
+
+```bash
+git add <filename>
+```
+
+The `<filename>` is the name and extension of the file to be added to the staging area. Multiple files can also be added by name with the following syntax:
+
+```bash
+git add <filename1> <filename2> <filename3>
+```
+
+If all files in the working directory need to be tracked, this syntax can be used, which should be run at the root directory:
+
+```bash
+git add .
+```
+
+Another command we can use to stage all changes is:
+
+```bash
+git add -A
+```
+
+Once a file has been added, `git status` can be run in the terminal/command prompt to check that the file is in the staging area. Git will show the changed files to be committed in <span style="color:green">green</span> text like so:
+
+![git-status-untracked-with-gitignore](git-add-and-status.png)
+
+More about the additional extensions to `git add` can be found in the [Add] section of the Codecademy Docs for Git.
```

Comment:
I would just embed the Docs entry link into the first mention of `git add` and delete this line.